tests/sources/test_url.py

```diff
@@ -129,6 +133,14 @@ def test_url_part_file_source():
         assert f.read() == b"GRIB7777GRIB7777"
 
 
+def test_url_netcdf_source_save():
```

Comment:
This is a big file to download. It is enough if you just load the `test.nc` example (it also a fieldlist) as a "file" and save it.